Desktop/components/JASP/Widgets/DataTableViewEdit.qml

```diff
@@ -194,8 +194,7 @@ Item
 
             onClicked: (mouse) =>
 			{
-				if(mouse.buttons & Qt.RightButton)
-					dataTableView.showPopupMenu(editItemRoot, mapToGlobal(mouse.x, mouse.y), rowIndex, columnIndex);
+				dataTableView.showPopupMenu(editItemRoot, mapToGlobal(mouse.x, mouse.y), rowIndex, columnIndex);
```

Comment:
```suggestion
				if(mouse.button === Qt.RightButton)
					dataTableView.showPopupMenu(editItemRoot, mapToGlobal(mouse.x, mouse.y), rowIndex, columnIndex);
```

Reply:
I think it‘s is unnecessary ? because:
```
acceptedButtons:	Qt.RightButton
```

Reply:
> I think it‘s is unnecessary ? because:
> 
> ```
> acceptedButtons:	Qt.RightButton
> ```

Oh you are right! I was too quick. But it's not bad either.